instagram-app/src/components/PostContainer/PostsPage.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PostContainer from './PostContainer'
+import SearchBar from '../SearchBar/SearchBar'
+
+import dummyData from '../../dummy-data';
+
+
+
+
+class PostsPage extends React.Component {
+  constructor(){
+    super();
+    this.state = {
+      dummy: [],
+      searchPosts: []
+    };
```

Comment:
It might help to use a name other than `dummy`, especially if you'll implement the ability to add/remove posts in the future. Perhaps `posts` might work as a clearer declarative name here.